### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Ibochkarev**
### What does it do?
**This update:**
- Improves interactivity
- Adds a new label with links to go
- Improves the external presentation of information
- Adds a description and some facts.

### Related issue(s)/PR(s)
NA

### CODE REVIEW

**Comment by zaigham** [comment]
I liked Emojis in other PR, I suggest to bring those in this PR.

File: README.md [hunk 1]
```diff
@@ -1,29 +1,46 @@
...
 ## Content Management System and Application Framework
 
 MODX lets you power anything from multi-language, multi-domain corporate sites to personal blogs to mobile APIs. Delivering true creative freedom and removing all restrictions, it lets you control the markup and design without having to code. You can also tailor its modular and extensible core to accommodate virtually any custom requirement or amount of traffic. MODX is the free open source software that meets your needs today—and tomorrow.
 
+MODX Revolution is the world’s fastest, most customizable Open Source PHP CMS. Your creative vision, no restrictions, no compromise.
+
+-   **50× Fewer Vulnerabilities** in government security databases
+-   **Millions** of Downloads
+-   **78 Versions Released** for relentless improvement
+-   Over **800** add-ons, **over 6 600 000** downloads, and an **uncountable** number of happy MODX-ers.
```
Comment (JoshuaLuckers) [inline README.md]: I don't think we should include information like statistics in the README. They change and require to many unnecessary changes over time.

**Comment by Ibochkarev** [comment]
> I liked Emojis in other PR, I suggest to bring those in this PR.

I like them too. Let's wait for the answers of other colleagues.

**Comment by JoshuaLuckers** [comment]
I am not convinced what use emoji in the README have.

**Comment by ghost** [comment]
In my opinion, for what it's worth, emojis would be a little unprofessional. A README should not be trying to be cute.

File: README.md (hunk 1 above)
Comment (Ibochkarev) [reply README.md]: Agree

**Comment by Mark-H** [comment]
Agreed with @sottwell about emojis. The badges are a nice touch. I'd 👍 this.

**Comment by Ibochkarev** [comment]
If everyone agrees with these changes, please confirm the changes and I will be able to publish this PR. I can't do that right now.

**Comment by JoshuaLuckers** [comment]
> If everyone agrees with these changes, please confirm the changes and I will be able to publish this PR. I can't do that right now.

You should be able to mark this PR ready for review. If not, I can.

**Comment by Ibochkarev** [comment]
Strangely, :unamused: the button was unavailable.

**Review by JoshuaLuckers — APPROVED** [review]

**Comment by JoshuaLuckers** [comment]
Sorry, closed the PR by mistake.

**Review by Mark-H — APPROVED** [review]
